tests/FSharp.Core.UnitTests/FSharp.Core/OperatorsModuleDynamic.fs

```diff
@@ -94,7 +94,10 @@ module OperatorsModuleDynamic =
         
         // Overflow
         let result = Operators.byte Single.MaxValue
-        Assert.AreEqual(0uy, result)
+        if not Info.isNetFramework then
+            Assert.AreEqual(255uy, result)
+        else
+            Assert.AreEqual(0uy, result)
```

Comment:
Why?

Reply:
It probably wouldn't hurt to add a comment like this to this file:

```fsharp
// For more information on the runtime-dependent behavior differences in this file, see:
// - https://github.com/dotnet/runtime/issues/61885
// - https://github.com/dotnet/runtime/pull/97529
```

Reply:
Yep, good idea, added